frontend/src/component/admin/users/UsersList/UsersList.tsx

```diff
@@ -356,7 +374,10 @@ const UsersList = () => {
                         }
                         elseShow={
                             <TablePlaceholder>
-                                No users available. Get started by adding one.
+                                <span data-loading>
```

Comment:
is it some extra cleanup you're doing here?

Reply:
yes. Quick fix to not show "no users available" when loading. This table needs proper placeholder loading, but that will be done in another PR